packages/SwingSet/test/test-xsnap-store.js

```diff
@@ -165,15 +165,15 @@ test('XS + SES snapshots are long-term deterministic', async t => {
   const h2 = await store.save(vat.snapshot);
   t.is(
     h2,
-    'b3bd291a9b42abb6acbe488a4da0c0eacee417a9f0ca94b88f7fbe4191bc43a0',
+    'a6024e911a9f10b7944beccf539722af0470c846254ef1959be89ab2254f9e6b',
```

Comment:
Why does this PR need to change these? When will other PRs need to do so?

Reply:
I see. Resolving here. But...

@dckc, in another PR, could we add a comment so anyone needing to change this understands what's going on? Thanks.